src/config/SSSDConfigTest.py

```diff
@@ -44,8 +44,8 @@ def create_temp_dir():
     return tempfile.mkdtemp(dir=test_dir)
 
 
-def striplist(l):
-    return([x.strip() for x in l])
+def striplist(domain_list):
```

Comment:
I suppose `striplist` acts as a utility function and is not always intended to operate on a domain_list. Perhaps it should be renamed `list` or something more generic.

Reply:
I've changed it to `the_list` because you never know when a common word like `list` will become a language keyword.